boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -9,6 +9,66 @@ document.addEventListener("DOMContentLoaded", () => {
   const loadMoreButton = document.getElementById("loadMoreButton");
   const categorySelect = document.getElementById("category");
   const categoryFieldSets = document.querySelectorAll("[data-category-wanted]");
+  const events = document.querySelectorAll('.event');
```

Comment:
why do you read all events here? this events-list will be out of date pretty with the next search/load more button and does not seem to get updated?

Reply:
done